code/modules/clothing/clothing.dm

```diff
@@ -92,6 +92,9 @@
 	if(W.get_sharpness() && cuttable)
 		if (alert(user, "Are you sure you want to cut the [src] into strips?", "Cut clothing:", "Yes", "No") != "Yes")
 			return
+		if(!src)
```

Comment:
um, no

just `qdel(src)` after cutting it into strips

Reply:
That is infact already done at the bottom of this proc

Reply:
ok then add a `if (QDELETED(src))` check at the very beginning not midway through

Reply:
It has to be after the alert else it won't work